pages/linux/tlp-stat.md

```diff
@@ -0,0 +1,15 @@
+# tlp-stat
+
+> TLP status reports. See also `tlp`.
+
+- Status report with configuration and all active settings:
```

Comment:
Perhaps this will read better?

```suggestion
- Generate status report with configuration and all active settings:
```

Reply:
Sure thing, just committed the update. Thanks!